oci_spec/src/caps_migrations.rs

```diff
@@ -0,0 +1,92 @@
+pub fn to_caps(local_cap: &LinuxCapabilityType) -> Capability {
+    match local_cap {
+        LinuxCapabilityType::CAP_CHOWN => Capability::CAP_CHOWN,
+        LinuxCapabilityType::CAP_DAC_OVERRIDE => Capability::CAP_DAC_OVERRIDE,
+        LinuxCapabilityType::CAP_DAC_READ_SEARCH => Capability::CAP_DAC_READ_SEARCH,
+        LinuxCapabilityType::CAP_FOWNER => Capability::CAP_FOWNER,
+        LinuxCapabilityType::CAP_FSETID => Capability::CAP_FSETID,
+        LinuxCapabilityType::CAP_KILL => Capability::CAP_KILL,
+        LinuxCapabilityType::CAP_SETGID => Capability::CAP_SETGID,
+        LinuxCapabilityType::CAP_SETUID => Capability::CAP_SETUID,
+        LinuxCapabilityType::CAP_SETPCAP => Capability::CAP_SETPCAP,
+        LinuxCapabilityType::CAP_LINUX_IMMUTABLE => Capability::CAP_LINUX_IMMUTABLE,
+        LinuxCapabilityType::CAP_NET_BIND_SERVICE => Capability::CAP_NET_BIND_SERVICE,
+        LinuxCapabilityType::CAP_NET_BROADCAST => Capability::CAP_NET_BROADCAST,
+        LinuxCapabilityType::CAP_NET_ADMIN => Capability::CAP_NET_ADMIN,
+        LinuxCapabilityType::CAP_NET_RAW => Capability::CAP_NET_RAW,
+        LinuxCapabilityType::CAP_IPC_LOCK => Capability::CAP_IPC_LOCK,
+        LinuxCapabilityType::CAP_IPC_OWNER => Capability::CAP_IPC_OWNER,
+        LinuxCapabilityType::CAP_SYS_MODULE => Capability::CAP_SYS_MODULE,
+        LinuxCapabilityType::CAP_SYS_RAWIO => Capability::CAP_SYS_RAWIO,
+        LinuxCapabilityType::CAP_SYS_CHROOT => Capability::CAP_SYS_CHROOT,
+        LinuxCapabilityType::CAP_SYS_PTRACE => Capability::CAP_SYS_PTRACE,
+        LinuxCapabilityType::CAP_SYS_PACCT => Capability::CAP_SYS_PACCT,
+        LinuxCapabilityType::CAP_SYS_ADMIN => Capability::CAP_SYS_ADMIN,
+        LinuxCapabilityType::CAP_SYS_BOOT => Capability::CAP_SYS_BOOT,
+        LinuxCapabilityType::CAP_SYS_NICE => Capability::CAP_SYS_NICE,
+        LinuxCapabilityType::CAP_SYS_RESOURCE => Capability::CAP_SYS_RESOURCE,
+        LinuxCapabilityType::CAP_SYS_TIME => Capability::CAP_SYS_TIME,
+        LinuxCapabilityType::CAP_SYS_TTY_CONFIG => Capability::CAP_SYS_TTY_CONFIG,
+        LinuxCapabilityType::CAP_SYSLOG => Capability::CAP_SYSLOG,
+        LinuxCapabilityType::CAP_MKNOD => Capability::CAP_MKNOD,
+        LinuxCapabilityType::CAP_LEASE => Capability::CAP_LEASE,
+        LinuxCapabilityType::CAP_AUDIT_WRITE => Capability::CAP_AUDIT_WRITE,
+        LinuxCapabilityType::CAP_AUDIT_CONTROL => Capability::CAP_AUDIT_CONTROL,
+        LinuxCapabilityType::CAP_AUDIT_READ => Capability::CAP_AUDIT_READ,
+        LinuxCapabilityType::CAP_SETFCAP => Capability::CAP_SETFCAP,
+        LinuxCapabilityType::CAP_MAC_OVERRIDE => Capability::CAP_MAC_OVERRIDE,
+        LinuxCapabilityType::CAP_MAC_ADMIN => Capability::CAP_MAC_ADMIN,
+        LinuxCapabilityType::CAP_WAKE_ALARM => Capability::CAP_WAKE_ALARM,
+        LinuxCapabilityType::CAP_BLOCK_SUSPEND => Capability::CAP_BLOCK_SUSPEND,
+        _ => panic!("not a Linux Capability Type"),
```

Comment:
You can use unreachable! here instead of panic! to communicate the intent a bit better. I am wondering though, is this even necessary? If you exhaustively check for all variants of the enum, a wildcard pattern should not be needed.

Reply:
Altered it a bit to be fully explicit :+1: . the caps dependency has a wildcard though